osu.Game/Rulesets/Mods/Mod.cs

```diff
@@ -131,22 +131,31 @@ public virtual string SettingDescription
         /// </summary>
         public virtual Mod CreateCopy()
         {
-            var copy = (Mod)Activator.CreateInstance(GetType());
+            var result = (Mod)Activator.CreateInstance(GetType(), true);
```

Comment:
Is the `true` argument required here? It seemed to work without before - I'm not sure we have mods with non-public parameterless ctors...

Reply:
that was also a drive-by change, was making a `MultiMod` private constructor as it had a public *parameterful* one, can revert